docs/sql-guide/functions/function-coalesce.md

```diff
@@ -0,0 +1,116 @@
+---
+title: COALESCE()
+layout: default
+parent: Functions
+grand_parent: SQL guide
+---
+
+# COALESCE() function
+
+`COALESCE()` function is used to return the first non-null value among the provided parameters.
```

Comment:
All the examples are based on values in a table.
Is this the only use case? If so it'd be good to give that context so the explanations that follow make better sense.

Reply:
The function is useful for
* Filling in default values when data is incomplete.
* Avoiding NULL values in calculations or concatenations.
* Providing a fallback value in case a preferred data point is not available.